### ISSUE TITLE
Overwriting word in lexicon

### ISSUE THREAD

**Issue by douglance** [spencermountain/compromise#736]
Is it possible to overwrite a word in the lexicon? 

For example, the word "includes" does not change tense correctly:

```
{
  PresentTense: "includs", 
  PastTense: "includded", 
  Gerund: "includding", 
  Infinitive: "includ", 
  FutureTense: "will includ"
}
```

I'd like to be able to overwrite this behavior with a custom word in the lexicon so it will translate properly.

**Comment by spencermountain** [comment]
hey Doug, good catch.
Yep, [changing the lexicon](https://observablehq.com/@spencermountain/compromise-lexicon) should be a straightforward thing. That conjugation looks like a bug, I'm happy to fix it.
thanks

**Comment by douglance** [comment]
Thanks! 

Is there a reference somewhere of the shape of all the words in the lexicon and how to overwrite them? It's not clear how I would overwrite each tense.

**Comment by spencermountain** [comment]
yeah, there's not a good resource for that. You can look at the lexicon-generation [here](https://github.com/spencermountain/compromise/blob/master/src/World/addWords.js) or print-out the whole lexicon with 
```js
nlp.world().words
```
cheers
